src/readii/analyze/plot_correlation.py

```
@@ -7,6 +7,9 @@
 from matplotlib.figure import Figure
 from scipy.linalg import issymmetric
 
+from readii.analyze.correlation import getSelfCorrelations
+from readii.io.writers.base_writer import BaseWriter
```

Ruff (0.8.2)

`readii.io.writers.base_writer.BaseWriter` imported but unused

Remove unused import: `readii.io.writers.base_writer.BaseWriter`

(F401)